src/api/app/views/webui/project/new.html.erb

```diff
@@ -1,52 +1,15 @@
-<% 
-unless @pagetitle
+<%
    @pagetitle = 'Create New Project'
-   @pagetitle = 'Create New Subproject' if @namespace
-end
 %>
 <% project_bread_crumb 'New Project' %>
 
 <h3><%= @pagetitle %></h3>
 <% unless User.current.is_admin? %>
   <p>
-    All users have a <b>home:&lt;login&gt;</b> project by default, and can also create new sub projects. New project outside of your home namespace can get requested via
-    <a href="https://bugzilla.opensuse.org/enter_bug.cgi?classification=7340&amp;product=openSUSE.org&amp;submit=Use+This+Product&amp;component=OBS%20Request">bugzilla</a>.
-    To discuss the requested project setup, please use the <a href="mailto:opensuse-buildservice@opensuse.org">opensuse-buildservice@opensuse.org</a> mailing list.
+    All users have a homes project by default. Yours is <%= link_to "home:#{User.current.login}", { controller: :project, action: :show,
+    project: "home:#{User.current.login}" } %>, and you can also
+    create sub projects inside. New projects outside of your home project can get
+    requested via <a href="mailto:<%= @configuration['admin_email'] %>">email</a>.
```

Comment:
> All users have a homes project by default. Yours is <%= link_to %>, and you can also
> create sub projects inside. New projects outside of your home project can get
> requested via &lt;a href="mailto:

All users have a home project, yours is <%= link_to USER %>, in that project you can also create subprojects. New projects outside of your home project can get requested via &lt;a href="mailto: